TESTING.md

```diff
@@ -109,6 +109,35 @@ mvn clean install -pl janusgraph-es -Delasticsearch.docker.image=elasticsearch
 mvn clean install -pl janusgraph-es -Delasticsearch.docker.version=1.5.1 -Delasticsearch.docker.image=elasticsearch
 ```
 
+### Running CQL Tests with a Cassandra
+
+**Note** Running Cql tests requires docker.
+
+```bash
+mvn clean install -pl janusgraph-cql -Pcassandra2
+mvn clean install -pl janusgraph-cql -Pcassandra3
+```
+
+Finally the `cassandra.docker.version` property can be used to test against arbitrary Cassandra versions. This is more complicated however because of differences across major versions in required server settings and Docker image names. The examples below illustrate the differences based on the Cassandra major version.
+
+```bash
+mvn clean install -pl janusgraph-cql -Dcassandra.docker.version=2.2.14
+mvn clean install -pl janusgraph-cql -Dcassandra.docker.image=cassandra
+mvn clean install -pl janusgraph-cql -Dcassandra.docker.version=3.11.2 -Dcassandra.docker.image=cassandra
+```
+
+#### ScyllaDB
+
+```bash
```

Comment:
Again, a short sentence to explain what this code block does would be good (same for the next section).